### PR TITLE
zulip_bots: Check Access Token on initialization for Mention bot.

### PR DESCRIPTION
**Pull request by ViRu-ThE-ViRuS**
Issue #171 
Updated `mention.py` -> added the required code.
Updated `mention.py` -> reformatted the existing code.
Updated `mention.conf` -> removed comment as some INI parsers don't support comments.

### CODE REVIEW

File: zulip_bots/zulip_bots/bots/mention/mention.conf [hunk 1]
```diff
@@ -1,3 +1,2 @@
 [mention]
 access_token = <access_token>
-# create an app here and get the access token from it: https://dev.mention.com/login
```
Comment (roberthoenig) [inline zulip_bots/zulip_bots/bots/mention/mention.conf]: This change should go into its own commit

File: zulip_bots/zulip_bots/bots/mention/mention.py [hunk 2]
```diff
@@ -36,8 +54,7 @@ def get_account_id(self) -> str:
             'Authorization': 'Bearer ' + self.access_token,
             'Accept-Version': '1.15',
         }
-        response = requests.get(
-            'https://api.mention.net/api/accounts/me', headers=get_ac_id_header)
+        response = requests.get('https://api.mention.net/api/accounts/me', headers=get_ac_id_header)
```
Comment (roberthoenig) [inline zulip_bots/zulip_bots/bots/mention/mention.py]: This change should go into its own commit.

File: zulip_bots/zulip_bots/bots/mention/mention.py [hunk 3]
```diff
@@ -2,13 +2,31 @@
...
+        test_query_header = {
+            'Authorization': 'Bearer ' + self.access_token,
+            'Accept-Version': '1.15',
+        }
+        test_query_response = requests.get('https://api.mention.net/api/accounts/me', headers=test_query_header)
+
+        try:
+            test_query_data = test_query_response.json()
+            if test_query_data['error'] == 'invalid_grant' and \
+               test_query_data['error_description'] == 'The access token provided is invalid.':
```
Comment (roberthoenig) [inline zulip_bots/zulip_bots/bots/mention/mention.py]: Are there cases where the error is "invalid_grant", but the error description is not "the access token provided is invalid"?

File: zulip_bots/zulip_bots/bots/mention/mention.py [hunk 4]
```diff
@@ -2,13 +2,31 @@
...
+                logging.error('Access Token Invalid. Please see doc.md to find out how to get it.')
```
Comment (roberthoenig) [inline zulip_bots/zulip_bots/bots/mention/mention.py]: You can add a sys.exit(1) here, since this shouldn't cause any problems as long as we do not cover check_access_token in tests.

File: zulip_bots/zulip_bots/bots/mention/mention.py (hunk 3 above)
Comment (ViRu-ThE-ViRuS) [reply zulip_bots/zulip_bots/bots/mention/mention.py]: I really didn't find any other errors in my testing or in the documentation with the error of 'invalid_grant'. I did this just to be sure that the reason for the error is the incorrect API Key.

**Comment by ViRu-ThE-ViRuS** [comment]
Done the requested changes. Thanks!

**Comment by roberthoenig** [comment]
Merged, after adding some detail to the second commit message. Thanks @ViRu-ThE-ViRuS! You won't see a "Merged" label on this branch because I merged your commits by checking them out in my own branch so I could modify the commit message. You can look in the commit history of python-zulip-api and find your commits there :)

**Comment by roberthoenig** [comment]
Whoops, closed the wrong one.

**Comment by roberthoenig** [comment]
Merging, thanks @ViRu-ThE-ViRuS!
